### PR TITLE
Multiple merges may happen in a minute

### PR DESCRIPTION
**Pull request by fejta**
See http://storage.googleapis.com/kubernetes-test-history/k8s-queue-health-2.svg for example output. Thanks for noticing this @lavalamp

### CODE REVIEW

File: queue-health/graph.py [hunk 1]
```diff
@@ -113,9 +113,12 @@ def render(history_lines, out_file):
         happy = online and not blocked
         happy_sum += happy
 
-        did_merge = last_merge and merged != last_merge
-        last_merge = merged
+        if merged > last_merge:
+            did_merge = merged - last_merge
+        else:  # Might be negative in case of restart
+            did_merge = 0
```
Comment (lavalamp) [inline queue-health/graph.py]: did_merge = merged

Comment (lavalamp) [inline queue-health/graph.py]: I guess your current code will just get it the next time around.

**Comment by rmmh** [comment]
What does this have to do with multiple merges in one minute?

**Comment by lavalamp** [comment]
@rmmh it fixes problems with both.

**Comment by rmmh** [comment]
oh, right, did_merge would be True or False before.

LGTM

**Comment by fejta** [comment]
Alright now handling the scenario where it restarts and merges before we poll

**Comment by lavalamp** [comment]
LGTM

**Pull request merged**